### PR TITLE
front-end changes to login page, home page & nav bar

### PR DESCRIPTION
**Pull request by Zalymo**
changed the nav bar to match the color palette we chose. adjusted the styling of the login page and got rid of the user info section on the home page. also made minor styling changes to the home page. will be making more front-end changes but I didn't want everything in one commit

### CODE REVIEW

File: osCam/static/core/login.css [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+    color: #222
+}
+
+#login-title {
+    position: relative;
+    top: -25%;
+}
+
+#label-form {
+    font-size: 20px;
```
[the remark this answers is not in the document]
Comment (Zalymo) [reply osCam/static/core/login.css:24]: I did show you guys the color scheme but I didn't share it. I'll post a link to it on the slack channel; however we may need a different color scheme with dark theme

File: osCam/static/core/dash.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 body {
-    background-color: rgb(0, 0, 0);
-    color: rgb(255, 255, 255);
+    background-color: rgb(255, 255, 255);
+    color: rgb(0, 0, 0);
```
[the remark this answers is not in the document]
Comment (Zalymo) [reply osCam/static/core/dash.css:3]: I agree, I'll find a new color scheme for dark theme

[the remark this answers is not in the document]
Comment (DJ-IRL) [reply osCam/static/core/dash.css]: looks great! Better than the plain white or black ever did!

**Comment by DJ-IRL** [comment]
this color change looks great, and makes the whole project feel more original.

**Pull request merged**